app/views/volume_mapping/show.html.haml

```diff
@@ -0,0 +1,7 @@
+- if %w(volume_mapping).include?(@display)
+  = render :partial => "layouts/gtl", :locals => {:action_url => "show/#{@record.id}"}
+- else
+  - if @showtype == "item"
```

Comment:
Is this a copy/pasted code or do we actually need this?

Reply:
copy/pasted. I assumed it was needed but seems that it works with just.
```
= render :partial => "layouts/textual_groups_generic"
```